test/Driver/unknown-inputs.swift

```diff
@@ -23,6 +23,13 @@
 // LINK-elf: 1: swift-autolink-extract, {0}, autolink
 // LINK-elf: 2: link, {0, 1}, image
 
+// LINK-SWIFTMODULES-macho: 0: input, "{{.*}}.swiftmodule", swiftmodule
+// LINK-SWIFTMODULES-macho: 1: link, {0}, image
+
+// LINK-SWIFTMODULES-elf: 0: input, "{{.*}}.swiftmodule", swiftmodule
+// LINK-SWIFTMODULES-elf: 1: swift-autolink-extract, {0}, autolink
```

Comment:
Ah, this is a great test because the behavior is wrong. :-) swift-autolink-extract should not be run on swiftmodules; the trouble is that I'm not sure whether it should *only* run on object files or also on, say, .a files. Let's say just object files for now.